doc/version-history.rst

```diff
@@ -8,6 +8,32 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+Cycle 37 revision 2
+===================
+
+* Update version of the following packages:
```

Comment:
Just a small comment: from the previous revisions I see that the "updated packages" section is always put as last bullet of the revision changelog. But this might not be an standard at all.